Documentation/architecture/net-platform-standard.md

```diff
@@ -134,13 +134,21 @@ Existing PCL projects in VS2013 and VS2015 (excluding UWP targets), can only tar
 
 ### Deprecated monikers
 
-| Platform | NuGet identifier |
-| ---------| --------------- |
-| ASP.NET 5.0 on .NET Framework | aspnet50 |
-| ASP.NET 5.0 on .NET Core | aspnetcore50 |
-| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 |
-| DNX on .NET Core 5.0 | dnxcore50  |
-| Windows 8 | winrt |
+| Platform | Deprecated NuGet identifier | Current NuGet identifier |
+| ---------| --------------------------- | ------------------------ |
+| ASP.NET 5.0 on .NET Framework | aspnet50 | net46 |
+| ASP.NET 5.0 on .NET Core | aspnetcore50 | netstandard1.3 |
+| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 | net451 - net46 |
+| DNX on .NET Core 5.0 | dnxcore50  | dotnet &rarr; dotnet5.4 &rarr; netstandard1.3 |
```

Comment:
> Anyone building portable-* can switch to using netstandard.

There are many libraries out there, whose portable TFM specify dnxcore50, that other developers probably want to use anyway.

I guess, that a separate paragraph is needed for the .NET Portable TFMs that contain dnxcore50. There might be TFMs, that contain dnxcore50, which target different netstandard versions (i.e. one portable TFM that targets 1.0, and another one that targets 1.1, both containing dnxcore5.0). So, is it bad practice to just do an `"imports": "dnxcore50"`? Should it better explicitly specify the full portable TFM?